templates/index.html

```diff
@@ -32,15 +32,15 @@
             <div id="arrowContainer">
               <h1>&lt;</h1>
             </div>
-            <span id="versionText" style="font-size: 20px;margin-top: 25px;">Version: 1.3.2</span>
+            <span id="versionText" style="font-size: 20px;margin-top: 25px;">Version: 1.4-beta</span>
```

Comment:
are we out of the beta yet?

Also should consider changing this to a variable, so that we edit a .conf file for version changes (making it server side)

Reply:
@CastyiGlitchxz made one but never finished implementing it. ill see whats needed to get it done and add it in.